strings-csharp/ReplaceLineBreaksInAStringCSharp/ReplaceLineBreaksInAStringCSharp/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Text.RegularExpressions;
+
+namespace ReplaceLineBreaksInAStringCSharp;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var stringReplaceResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheStringReplaceMethod();
+        Console.WriteLine(stringReplaceResult);
+        
+        var stringReplaceLineEndingsResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheStringReplaceLineEndingsMethod();
+        Console.WriteLine(stringReplaceLineEndingsResult);
+        
+        var regularExpressionReplaceResult = new ReplaceLineBrake().ReplaceLineBreaksUsingTheRegularExpressionReplaceMethod();
+        Console.WriteLine(regularExpressionReplaceResult);
+    }
+}
+
+public class ReplaceLineBrake
+{
+    public string ReplaceLineBreaksUsingTheStringReplaceMethod()
+    {
+        var text = "This is a line.\nThis is another line.";
```

Comment:
This comment applies across these methods. We should change the method signature to accept a string and then pass in the strings that we are going to replace the endings on. This will make the sample code clearer as well as the unit tests. We can then, in the unit tests, verify that the method does not modify strings that do not contain newlines.

Reply:
The methods were just meant as a grouping for the code. Changing the method signature to accept the input text complicates the code.